src/Controller/ModulesController.php

```diff
@@ -460,9 +459,10 @@ public function relationshipsJson($id, string $relation) : void
                     $available = '/folders';
                     break;
                 default:
-                    $response = $this->apiClient->get($path, ['page_size' => 1]); // page_size 1: we need just the available link from response
+                    $response = $this->apiClient->get($path, $this->request->getQueryParams());// ['page_size' => 1]); // page_size 1: we need just the available
```

Comment:
I would remove the comment, because it's unuseful with new code (it was fine to explain why the filter was `'page_size' => 1`)

Reply:
i actually wanted to undo this change thanks for spotting it